pom.xml

```diff
@@ -77,7 +77,7 @@
     <javaVersion>1.8</javaVersion>
     <javaTestVersion>${javaVersion}</javaTestVersion>
 
-    <mavenVersion>3.5.0</mavenVersion>
```

Comment:
Why this change?

Reply:
No problem. I didn't mean to leave this in. I was just checking if the tests passed regardless of the version used.

Reply:
Ok